README.md

```diff
@@ -1,30 +1,66 @@
 # elabforms
 A set of tools to create and manage standardized forms for eLabFTW
 
+## Create a virtual environment
+```bash
+python3 -m venv myenv
+source myenv/bin/activate
+```
 
 ## Installation
+Make sure you have `git` and `pip` installed. You can download and unzip the following repositories:
+- git@github.com:INT-NIT/elabforms_INTProjects.git
```

Comment:
this line should be removed... this is the general documentation on how to use elabforms, it should not point to a private repository!